src/content/en/updates/2016/12/imagecapture.md

```diff
@@ -0,0 +1,349 @@
+
+* Tap or click the **GRAB FRAME** button to draw a frame from the video on a
+`<canvas>`.
+
+* Tap or click the **TAKE PHOTO** button to call `takePhoto()` and display the
+result in an `<img>`.
+
+* If the camera you are using supports zoom, a zoom slider will be displayed.
+
+Several variables from the demo are available from the console:
+
+* **constraints**
+* **imageCapture**
+* **mediaStream**
+
+<div id="ic-demo">
+  <button id="grabFrame">Grab Frame</button>
+  <button id="takePhoto">Take Photo</button>
+  <div class="select">
+    <label for="videoSource">Video source: </label><select id="videoSource"></select>
+  </div>
+  <input class="hidden" id="zoom" type="range" step="20">
+  <video autoplay class="hidden"></video>
+  <canvas class="hidden"></canvas>
+  <img class="hidden">
+</div>
+
+## Camera Capabilities
+
+In the demo above, you'll notice a difference in dimensions between the
+`grabFrame()` and `takePhoto()` results.
+
+The `takePhoto()` method gives access to the camera's maximum resolution.
+
+`grabFrame()` just takes the next-available `VideoFrame` in the `MediaStream`
+inside the renderer process, whereas `takePhoto()` interrupts the `MediaStream`,
+reconfigures the camera, takes the photo (usually in a compressed format,
+hence the `Blob`) and then resumes the `MediaStream`. In essence, this means
+that `takePhoto()` can give access to the full still-image resolution
+capabilities of the camera. Previously, it was only possible to 'take a photo' by
+calling `drawImage()` on a `canvas` element, using a video as the source (as per the
+example [here](https://webrtc.github.io/samples/src/content/getusermedia/canvas/)).
+
+In this demo, the `<canvas>` dimensions are set to the resolution of the video
+stream, whereas the (natural) size of the `<img>` is the maximum still-image
+resolution of the camera — though, of course, CSS is used to set the display
+size of both.
+
+The full range of possible camera resolutions for still images can be accessed
+via the `MediaSettingsRange` values for `PhotoCapabilities.imageHeight` and
+`imageWidth`. Note that the minimum and maximum width and height constraints for
+`getUserMedia()` are for video, which (as discussed) may be different from the
+camera capabilities for still images. In other words, you may not be able to
+access the full resolution capabilities of your device when saving from
+`getUserMedia()` to a canvas. The WebRTC [resolution constraint
+demo](https://webrtc.github.io/samples/src/content/getusermedia/resolution)
+shows how to set `getUserMedia()` constraints for resolution.
+
+## Anything Else?
+
+* The [**Shape Detection API**](https://www.chromestatus.com/feature/4757990523535360) works well with Image Capture: call `grabFrame()`
+repeatedly to feed `ImageBitmap`s to a `FaceDetector` or `BarcodeDetector`. Find
+out more about the API from Paul Kinlan's [blog post](https://paul.kinlan.me/face-detection/).
+
+* The **Camera flash** (torch) can be accessed via
```

Comment:
Need some language finessing here. Torches are called flashlights in North America. Maybe just call it 'the light'?